awscli/alias.py

```diff
@@ -356,9 +384,9 @@ def __call__(self, args, parsed_globals):
             # a built-in command.
             if shadow_name == parsed_alias_args.command:
                 LOG.debug(
-                    'Using shadowed command object: %s '
-                    'for alias: %s', self._shadow_proxy_command,
-                    self._alias_name
+                    'Using shadowed command object: %s ' 'for alias: %s',
```

Comment:
This is a log statement, the % syntax is fine, we just want to remove the extraneous quotes and whitespace from the line collapse.
```suggestion
                    'Using shadowed command object: %s for alias: %s',
```